google/cloud/bigtable/tests/admin_async_integration_test.cc

```diff
@@ -230,7 +230,7 @@ TEST_F(AdminAsyncIntegrationTest, AsyncDropRowsByPrefixTest) {
 
   promise_drop_row.get_future().get();
   auto actual_cells = ReadRows(table, bigtable::Filter::PassAllFilter());
-  EXPECT_STATUS_OK(DeleteTable(table_id));
+  EXPECT_STATUS_OK(table_admin_->DeleteTable(table_id));
```

Comment:
I think this test could use `GetTable()`.

Reply:
Done.